lib/nightmare.js

```diff
@@ -80,8 +80,9 @@ function Nightmare(options) {
 
   process.setMaxListeners(Infinity);
   process.on('uncaughtException', function(err) {
-    console.error(err.stack);
+    debug(err.stack);
```

Comment:
Since this is handling the exception in the same process as a user’s code, shouldn’t we be using `console.error` instead of `debug` so that we don’t swallow stack traces originating from their code? Or am I misreading this?